_posts/2022-02-16-solidity-0.8.12-release-announcement.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.12 Release Announcement'
+date: '2022-02-16'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.12](https://github.com/ethereum/solidity/releases/tag/v0.8.12)
+improves the javascript/wasm binary and fixes several bugs.
+
+## Notable New Features
+
+### Emscripten build / solcjs
+
+The code for the javascript wrappers of the compiler has been partially
+ported to TypeScript, which also impacts some of the aspects of the npm package.
```

Comment:
To be precise, it should actually not impact the npm package at all (other than now it contains JS compiled from TS). You should not have to change your imports for example.

What it impacts is the package build process (but that's on us) and anyone using the repo directly rather than the package.